recipes/dbus/1.x.x/test_package/CMakeLists.txt

```diff
@@ -0,0 +1,10 @@
+cmake_minimum_required(VERSION 3.1)
+project(PackageTest)
+
+include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
+conan_basic_setup()
+
+find_package(DBus1 REQUIRED)
```

Comment:
I think it might be work testing it with the pkg_config method https://stackoverflow.com/a/38520201/8480874